metron-interface/metron-rest/src/main/java/org/apache/metron/rest/service/impl/StormCLIWrapper.java

```diff
@@ -75,37 +81,50 @@ public int stopIndexingTopology(boolean stopNow) throws RestException {
   protected int runCommand(String[] command) throws RestException {
     ProcessBuilder pb = getProcessBuilder(command);
     pb.inheritIO();
-    Process process = null;
+    LOG.debug("Running command: cmd={}", String.join(" ", command));
+
+    Process process;
     try {
       process = pb.start();
       process.waitFor();
+
     } catch (Exception e) {
       throw new RestException(e);
     }
-    return process.exitValue();
+
+    int exitValue = process.exitValue();
+    LOG.debug("Command completed: cmd={}, exit={}", String.join(" ", command), exitValue);
+
+    return exitValue;
   }
 
   protected String[] getParserStartCommand(String name) {
-    String[] command = new String[7];
+    String[] command = new String[9];
     command[0] = environment.getProperty(MetronRestConstants.PARSER_SCRIPT_PATH_SPRING_PROPERTY);
     command[1] = "-k";
     command[2] = environment.getProperty(MetronRestConstants.KAFKA_BROKER_URL_SPRING_PROPERTY);
     command[3] = "-z";
     command[4] = environment.getProperty(MetronRestConstants.ZK_URL_SPRING_PROPERTY);
     command[5] = "-s";
     command[6] = name;
+    command[7] = "-ksp";
+    command[8] = environment.getProperty(MetronRestConstants.KAFKA_SECURITY_PROTOCOL_SPRING_PROPERTY);
     return command;
   }
 
   protected String[] getEnrichmentStartCommand() {
-    String[] command = new String[1];
+    String[] command = new String[3];
```

Comment:
I don't think the -ksp flag is needed for the enrichment or indexing topologies.  This setting comes from enrichment.properties and elasticsearch.properties.